### PR TITLE
Map HeldItemRenderer

### PR DESCRIPTION
**Pull request by matthewperiut**

### CODE REVIEW

File: mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping [hunk 1]
```diff
@@ -1 +1,25 @@
 CLASS net/minecraft/class_556 net/minecraft/client/render/item/HeldItemRenderer
+	FIELD field_2401 minecraft Lnet/minecraft/client/Minecraft;
+	FIELD field_2402 stack Lnet/minecraft/class_31;
+	FIELD field_2403 height F
+	FIELD field_2404 prevHeight F
+	FIELD field_2405 blockRenderManager Lnet/minecraft/class_13;
+	FIELD field_2406 itemRenderer Lnet/minecraft/class_321;
```
Comment (DanyGames2014) [inline mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping]: Might wanna name this mapItemRenderer

File: mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping [hunk 2]
```diff
@@ -1 +1,25 @@
 CLASS net/minecraft/class_556 net/minecraft/client/render/item/HeldItemRenderer
```
Comment (DanyGames2014) [inline mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping:1]: Please map the Minecraft argument in constructor for the sake of completeness

**Review by DanyGames2014 — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping (hunk 1 above)
Comment (matthewperiut) [reply mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping]: https://maven.fabricmc.net/docs/yarn-1.21+build.6/net/minecraft/client/render/MapRenderer.MapTexture.html
ty dany

File: mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping (hunk 2 above)
Comment (matthewperiut) [reply mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping:1]: Will do

**Pull request merged**